exercises/simple-cipher/src/test/java/SimpleCipherStepOneTest.java

```diff
@@ -21,8 +24,10 @@ public void setup() {
      */
     @Test
     public void cipherCanEncode() {
+        String plainText = "aaaaaaaaaa";
         String cipherText = cipherWithDefaultKey.getKey().substring(0, 10);
         assertEquals(cipherText, cipherWithDefaultKey.encode("aaaaaaaaaa"));
+        assertEquals(cipherText, cipherWithDefaultKey.encode(plainText));
```

Comment:
You don't need both assert equals statements 🙂

Reply:
My bad